packages/talker_dio_logger/lib/talker_dio_logger_settings.dart

```diff
@@ -93,6 +94,10 @@ class TalkerDioLoggerSettings {
   /// You can add your custom logic to log only specific Dio error [DioException].
   final bool Function(DioException response)? errorFilter;
 
+  /// Header values for the specified keys in the Set will be replaced with *****.
+  /// Case insensitive
+  final Set<String> hideHeaderValuesForKeys;
```

Comment:
``hideHeaderValuesForKeys`` is huge and hard name for this field 
Can we rename it to ``hiddenHeaders`` ?